.github/CODEOWNERS

```diff
@@ -1170,6 +1170,9 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 /x-pack/test/functional/apps/infra @elastic/obs-ux-infra_services-team
 /x-pack/test/functional/apps/infra/logs @elastic/obs-ux-logs-team
 
+# Observability
+/x-pack/test/functional/es_archives/cases/default @elastic/observability
```

Comment:
Looks like there's no `elastic/observability` team. Based on the path, were you meaning assign ownership to `elastic/response-ops` or maybe `elastic/observability-ui`?

Reply:
Ah, good catch!

Reply:
https://github.com/elastic/kibana/pull/194411/commits/8820f0721a86f3616d941bacf432db4589877b16 due to the one use [here](https://github.com/elastic/kibana/blob/20dcf5f7211c6e8ed166fc7bb9a61a733384a529/x-pack/test/observability_functional/apps/observability/feature_controls/observability_security.ts#L29) and that test file's assignment [here](https://github.com/elastic/kibana/blob/20dcf5f7211c6e8ed166fc7bb9a61a733384a529/.github/CODEOWNERS#L1175)